spec/models/item_spec.rb

```diff
@@ -5,4 +5,14 @@
     expect(create(:item)).to be_valid
   end
 
+  it 'has a status of OOS if condition is OOS' do
+    item = build :item, condition: 'Out of Service', status: 'Unassigned'
+    expect(item.status).to eq('Out of Service')
```

Comment:
It is testing that an item that is not in a serviceable condition, can not be assigned to a task.

https://github.com/kgf/ReadyResponder/blob/add_condition_to_item_and_ui_inprovements/app/models/item.rb#L39